src/ng/http.js

```diff
@@ -71,14 +71,16 @@ function headersGetter(headers) {
   var headersObj = isObject(headers) ? headers : undefined;
 
   return function(name) {
+    var existingHeaderName, lowercaseName = lowercase(name);
```

Comment:
Hmm, I would rather not as if / for don't create lexical scope in JS. I know that browsers will do it for you but I think that most of the code in AngularJS is declaring variables at the lexical scope, no?

Anyway, I don't feel strong about it, let's just be consistent.